package.json

```diff
@@ -11,8 +11,8 @@
     "jest": "^29.4.3",
     "jest-environment-jsdom": "^29.4.3",
     "parcel": "^2.8.3",
-    "react": "^18.3.0-next-b72ed698f-20230303",
-    "react-dom": "^18.3.0-next-b72ed698f-20230303",
+    "react": "https://react-builds.vercel.app/api/prs/26317/packages/react",
+    "react-dom": "https://react-builds.vercel.app/api/prs/26317/packages/react-dom",
```

Comment:
That's a neat trick. Doesn't seem to work with `npm install` though. Guess it's a Yarn specific thing?

Sure enough though, this version does fix the bug report I shared!

Going to play with it more now to see if it also fixes the original thing I was wrestling with!

Reply:
I think it works if you do `react@https://react-builds.vercel.app/api/prs/26317/packages/react`

Reply:
Yup! It works with Yarn. I was just using NPM by itself and noticed the install failed.

Reply:
I'll go ahead and publish an actual `@experimental` release once CI finishes, too.